<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RangeParameterValueForm</class>
 <widget class="QWidget" name="RangeParameterValueForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="minLabel">
     <property name="text">
      <string>Min:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="minLineEdit"/>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="maxLabel">
     <property name="text">
      <string>Max:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLineEdit" name="maxLineEdit"/>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="recommendationLabel">
     <property name="text">
      <string>Recommendation:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="3">
    <widget class="QTextEdit" name="recommendationTextEdit"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
